docs/Changelog

```diff
@@ -37,6 +44,7 @@ Stable versions
 	- Allow xmp_smix_play functions to play key off, cut and fade events.
 	- Integrate the OXM depacker with the XM loader.
```

Comment:
@ccawley2011 merged the OXM depacker into the XM loader.

Reply:
Fixed